core/src/main/java/edu/wpi/grip/core/serialization/Project.java

```diff
@@ -59,6 +59,7 @@ public void initialize(StepConverter stepConverter,
       cp.getAllClasses()
           .stream()
           .filter(ci -> ci.getPackageName().startsWith("edu.wpi.grip"))
+          .filter(ci -> !ci.getName().contains("BeanInfo")) // No BeanInfo classes in embedded JREs
```

Comment:
Can `BeanInfo` be moved into the `UI` project instead?
Keep the same package, just move it to the other project.